src/features/todos.ts

```diff
@@ -1,8 +1,14 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    setTodos: (_state: Todo[], action: PayloadAction<Todo[]>) => {
+      return action.payload;
```

Comment:
Here you’re correctly returning the action.payload directly, which is fine in this case since it's replacing the entire state 👍